docs/conf.py

```diff
@@ -27,10 +27,12 @@
 
 from datetime import datetime
 import os
-import sys
+import sys; sys.setrecursionlimit(1500)
```

Comment:
Nitpick: Unlike C, semicolon is not encouraged in Python. Concatenating multiple commands in one line makes it hard to debug.
```suggestion
import sys

sys.setrecursionlimit(1500)
```